docker_autoscaler.tf

```diff
@@ -103,12 +103,39 @@ resource "aws_autoscaling_group" "autoscaler" {
           launch_template_id = aws_launch_template.this[0].id
           version            = aws_launch_template.this[0].latest_version
         }
+
         dynamic "override" {
           for_each = var.runner_worker_docker_autoscaler_asg.types
           content {
             instance_type = override.value
           }
         }
+
+        dynamic "override" {
+          for_each = var.runner_worker_docker_autoscaler_asg.instance_requirements
+          content {
+            instance_requirements {
```

Comment:
praise: Gosh. This is a game changer! I already have troubles to get an available instance. This makes it a lot easier to select multiple instance types.